### ISSUE TITLE
Fine tune error

### ISSUE THREAD

**Issue by madhushankara** [wickaknight/LLM-fine-tune-with-doc#2]
===== TRAINING LOSS ANALYSIS =====
No TensorBoard logs found. Looking for alternative logs...
No training logs found. Manual loss calculation required.
To track training loss in future runs, add 'logging_dir="./logs"' to your TrainingArguments.

===== PARAMETER STATISTICS =====
LoRA Configuration:
  Rank (r): 8
  Alpha: 32
  Dropout: 0.1
  Target modules: ['v_proj', 'k_proj', 'q_proj', 'o_proj']
Loading model to analyze parameters...
Error analyzing parameters: Weights only load failed. This file can still be loaded, to do so you have two options, do those steps only if you trust the source of the checkpoint.
